homeassistant/components/yeelight/light.py

```diff
@@ -2,8 +2,25 @@
 import logging
 
 import voluptuous as vol
+import yeelight
 from yeelight import RGBTransition, SleepTransition, Flow, BulbException
 from yeelight.enums import PowerMode, LightType, BulbType, SceneClass
+from yeelight.transitions import (
```

Comment:
Maybe it would be safer to make
``` from yeelight import transitions``` and then in transitions method use transitions.disco etc, I'm worried that names will collide with something, for example rgb

Reply:
Yep, and the pipe is failing because of this.

Or rename all transitions to `yee[transition_name]`.

Reply:
Done

Reply:
> W0621: Redefining name 'transitions' from outer scope (line 6) (redefined-outer-name)

Not enough to fix the pipe, will call it `yee_transitions`

Reply:
Should be nice now 😉